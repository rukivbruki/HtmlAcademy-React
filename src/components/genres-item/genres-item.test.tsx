import * as React from "react";
import * as renderer from "react-test-renderer";
import GenresItem from "./genres-item";

it(`GenresItem renders correctly`, () => {
  const tree = renderer
    .create(
        <GenresItem
          genre={`test`}
          onClick={() => void 0} />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
