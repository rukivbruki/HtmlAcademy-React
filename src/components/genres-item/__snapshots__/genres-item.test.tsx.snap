// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GenresItem renders correctly 1`] = `
<li
  className="catalog__genres-item undefined"
>
  <a
    className="catalog__genres-link"
    href="#"
    onClick={[Function]}
  >
    test
  </a>
</li>
`;
